gate-web/gate-web.gradle

```diff
@@ -45,8 +45,11 @@ dependencies {
 
   // SAML
   compile('org.opensaml:opensaml:2.6.4')
-  compile("org.springframework.security.extensions:spring-security-saml2-core:1.0.2.RELEASE")
-  compile("org.springframework.security.extensions:spring-security-saml-dsl:1.0.0.M3") {
+  compile('ca.juliusdavies:not-yet-commons-ssl:0.3.11')
```

Comment:
According to https://github.com/spring-projects/spring-security-saml/issues/263 the 1.0.4 `saml-core` bumped up to `not-yet-commons-ssl:0.3.17` ... I've not pulled this PR locally and looked specifically at the transitive dependency changes but does this make sense?

Are we really saying that we need an older `not-yet-commons-ssl` to support this?

I'm not able to find any source for `not-yet-commons-ssl` and am somewhat reluctant to arbitrarily start swapping dependencies for an abandonware lib.

Reply:
On that note (abandonware libs)... the `opensaml` v2 dependency from the line above was EOL'd 2 years ago (see https://wiki.shibboleth.net/confluence/display/OpenSAML/Home). That's probably going to need to be looked at as well.